src/lib/webpack/webpack-base-config.js

```diff
@@ -97,6 +100,37 @@ export default (env) => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
+									useCache: true,
+									configFileName: tsconfig,
```

Comment:
@rkostrzewski yep.

@porfirioribeiro CSS-Modules should work out of the box. The only difference is that you'll have to revert to `require` statements:

```ts
// styles appears as type any, because ts has no idea of `.css` files
const styles = require("foo.css");
```